upf/test/test_upf.py

```diff
@@ -359,7 +367,6 @@ def associate(self):
             IE_NodeId(id_type="FQDN", id="ergw")
         ]), PFCPAssociationSetupResponse)
         self.assertEqual(CauseValues[resp[IE_Cause].cause], "Request accepted")
-        self.assertIn(b"vpp", resp[IE_EnterpriseSpecific].data)
```

Comment:
Is this change somehow related to this PR? This is actually a problem (#104), as the build id should contain `upg` not `vpp`, but I thought of fixing it separately and not by removing this check, but rather by fixing the id itself and replacing `vpp` with `upg` here

Reply:
In general, association setup response checks should be enhanced with:
- proper build ID check
- BBF UserPlane Features checks
I think it's worth a separate review